.circleci/main.yml

```diff
@@ -845,3 +888,76 @@ workflows:
           filters: *only-release-tags
           requires:
             - sign-off
+  nightly-release:
```

Comment:
Is the only difference between the `nightly-releases` workflow and the regular releases workflow is the `approval` step? If so, does the `filter` section in sign-off mean it will only run on the regular release tags? Alternatively, does adding a `when` to just the approval step work?

Reply:
yea thats the only difference. and no i've tried that and it just ignores it. plus even if it worked you would need to modify the requires down stream to conditionally wait on approval or not. so just easier to do a whole other pipeline definition.

Reply:
That's good to go this way, perhaps down the road we can use dynamic config for this.